recipe/meta.yaml

```diff
@@ -51,24 +51,22 @@ requirements:
     - pybind11 <2.11
     - pcre
     - gflags
-    - cyrus-sasl =2.1.27
     - msgpack-c
     - aws-sdk-cpp
     - azure-core-cpp
     - azure-identity-cpp
     - azure-storage-blobs-cpp
     - bitmagic
-    - krb5 =1.20.1
     - xxhash
     - semimap
     - grpcio-tools
     - protobuf
     - spdlog
     - robin_hood
     - recycle
-    - lmdb ==0.9.22
-    - lmdbxx ==0.9.14
-    - fmt <10
+    - lmdb
```

Comment:
Note to reviewers: currently, this makes ArcticDB use LMDB 0.9.29, which causes 1 parametrized tests out of 5000 to fail upstream.

See https://github.com/man-group/ArcticDB/pull/437.